Resources/public/js/models/ez-locationmodel.js

```diff
@@ -103,6 +103,22 @@ YUI.add('ez-locationmodel', function (Y) {
         },
 
         /**
+         * Updates the sortfield and sortOrder of the location
+         *
+         * @method _updateSorting
+         * @param {Object} options the required for the update
+         * @param {Object} options.api (required) the JS REST client instance
+         * @param {Function} callback a callback executed when the operation is finished
+         */
+        updateSorting: function (options, sortField, sortOrder, callback) {
+            var locationUpdateStruct = options.api.getContentService().newLocationUpdateStruct();
+
+            locationUpdateStruct.body.LocationUpdate.sortField = sortField;
+            locationUpdateStruct.body.LocationUpdate.sortOrder = sortOrder;
+            options.api.getContentService().updateLocation(this.get('id'), locationUpdateStruct, callback);
```

Comment:
in case of success, you should update the `sortField` and `sortOrder` of the Location.